Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:14.04
```

Comment:
It's not. However Docker uses the host OS' kernel. So depending upon what
you host this container in you'll get that version of  the kernel.

On Mon, Mar 7, 2016 at 10:54 AM, Jonathan Salwan <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/JonathanSalwan/Triton/pull/307#discussion_r55222914>:
>
> > @@ -0,0 +1,31 @@
> > +FROM ubuntu:14.04
>
> Is ubuntu 14.04 using a kernel 4.x ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JonathanSalwan/Triton/pull/307/files#r55222914>.
>